src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -92,27 +92,18 @@ private void initializeGroups(Set<? extends Tag> tagsToAdd) {
     @Override
     public ViewCommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-
-        //        if (this.tagType == TagType.GROUP) {
-        //            personToEdit.addGroupTags(this.groupTags);
-        //            if (personToEdit instanceof User) {
-        //                return new ViewCommandResult(String.format(MESSAGE_GROUP_TAG_USER_SUCCESS
-        //                        + "Name: " + personToEdit.getName().toString() + '\n'
-        //                        + "Groups: " + personToEdit.getImmutableGroupTags().toString()), personToEdit);
-        //            }
-        //            return new ViewCommandResult(String.format(MESSAGE_GROUP_TAG_PERSON_SUCCESS
-        //                    + "Name: " + personToEdit.getName().toString() + '\n'
-        //                    + "Groups: " + personToEdit.getImmutableGroupTags().toString()), personToEdit);
-        //        }
+        if (this.tagType == TagType.GROUP) {
+            return tagGroups(model);
```

Comment:
Ideally, the logic for this part should be the same as the modules part. This is so that you don’t need to check instanceof and type cast later on.